app/v4/controllers/RetrieveCalculationController.scala

```diff
@@ -63,6 +65,14 @@ class RetrieveCalculationController @Inject() (val authService: EnrolmentsAuthSe
         RequestHandler
           .withValidator(validator)
           .withService(service.retrieveCalculation)
+          .withAuditing(AuditHandler(
+            auditService,
+            auditType = "RetrieveATaxCalculation",
+            transactionName = "retrieve-a-tax-calculation",
+            apiVersion = Version.from(request, orElse = Version4),
+            params = Map("nino" -> nino, "calculationId" -> calculationId, "taxYear" -> taxYear),
+            None
```

Comment:
So that the response is included in the audit (the tests should fail if this is not done).
```suggestion
            includeResponse = true
```

Likewise for v5.